gitTagMaster/gitTagMaster

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# Author: Tina Barfield https://github.com/tinabme
+
+##########################################################################################
+# Script to simplify git tagging of multiple repos for release
+#
+# Usage: script name then each repo separated by space then tag 
+# Notes: 
+#     - You may want to run the first time with #git tag $TAG & #git push --tags commented out
+#       then when you have all of the proper repos uncomment and run for real
+#     - You can use default hardcoded repos or pass repos as arguments
+# 
+# Examples:
+#   From parent directory in which each of the repo directories live
+#     - with repo arguments: ./gitTagMaster.sh mySiteUI_js mySiteLambda mySiteEngine_ruby mySiteCLI v2.1.3
+#     - without repo arguments: ./gitTagMaster.sh v2.1.3
+##########################################################################################
+
+# check if stdout is a terminal...
+if test -t 1; then
+    # see if it supports colors...
+    ncolors=$(tput colors)
+
+    if test -n "$ncolors" && test $ncolors -ge 8; then
+        bold="$(tput bold)"
+        underline="$(tput smul)"
+        standout="$(tput smso)"
+        normal="$(tput sgr0)"
+        black="$(tput setaf 0)"
+        red="$(tput setaf 1)"
+        green="$(tput setaf 2)"
+        yellow="$(tput setaf 3)"
+        blue="$(tput setaf 4)"
+        magenta="$(tput setaf 5)"
+        cyan="$(tput setaf 6)"
+        white="$(tput setaf 7)"
+    fi
+fi
+
+# verify that the tag argument is passed and the last argument starts with a v
+if [[ $# -gt 0 ]] && [[ ${!#} == v* ]]; then
+
+  # default hardcoded repos, or passed repo arguments
+  if [ $# -eq 1 ]; then
+    REPOS="mySiteUI_js
```

Comment:
Assume this was for yourself, in our case it does not make sense to have hardcoded repos